launchable/commands/record/commit.py

```diff
@@ -55,21 +55,35 @@ def exec_jar(source, max_days, dry_run):
 
     base_url = get_base_url()
 
+    command = [java]
     https_proxy = os.getenv("HTTPS_PROXY")
     proxy_option = _build_proxy_option(https_proxy) if https_proxy else ""
-
-    subprocess.check_output("{java} {proxy_option} -jar \"{jar_file_path}\" ingest:commit -endpoint {endpoint} -max-days {max_days} {audit} {scrub_pli} {dry_run} {source}"
-                            .format(
-                                java=java,
-                                proxy_option=proxy_option,
-                                jar_file_path=jar_file_path,
-                                endpoint="{}/intake/".format(base_url),
-                                max_days=max_days,
-                                audit="-audit" if Logger().logger.isEnabledFor(LOG_LEVEL_AUDIT) else "",
-                                scrub_pli="-scrub-pii",
-                                dry_run="-dry-run" if dry_run else "",
-                                source=source),
-                            shell=True)
+    if proxy_option != "":
+        command.append(proxy_option)
+
+    command.extend(
```

Comment:
How about adding a comment that why we use extend and don't use system?

Reply:
what do you mean by system?

Reply:
added comment.
https://github.com/launchableinc/cli/pull/393/commits/d5ec7704fce3b154eb11bc3bb3a46c09a4cffc92

Reply:
I wanted to say python method's `extend` and `os. system.`

Reply:
`extend` is just adding a list to another list, and it is different topic of `os.system`.

E.g.
```python
a_list = [1, 2, 3]
b_list = [4, 5, 6]
a_list.extend(b_list)
print(a_list)
# [1, 2, 3, 4, 5, 6]
```

Reply:
Sounds good
I thought we should comment on why we build a command step by step, why not write one-liner. And your comment meets it. Thank you